.circleci/config.yml

```diff
@@ -82,5 +82,5 @@ workflows:
   commit:
     jobs:
       - test-js-ipfs
-      - test-go-ipfs
+      #- test-go-ipfs # @todo Determine why go-ipfs is seemingly not available in CI on 5001
```

Comment:
It's weird, in the other PRs created by the renovate bot it works

Reply:
@rdig mentioned that this might not be working? But as you say... if those passed 🤔

Reply:
Let's try to reactivate those and see what happens

Reply:
Ok, will do 👍